homework_create_more_models/README.md

```diff
@@ -0,0 +1,269 @@
+# Homework: create comment model
+
+Now we only have Post model, what about receiving some feedback from your readers?
+
+## Creating comment blog model
+
+Let's open `blog/models.py` and append this piece of code to the end of file:
+
+    class Comment(models.Model):
+    	post = models.ForeignKey('blog.Post', related_name='comments')
+    	author = models.CharField(max_length=200)
+    	text = models.TextField()
+    	created_date = models.DateTimeField(default=timezone.now)
```

Comment:
Hmm, you're right, but timezone.now is used in django girls tutorial here http://tutorial.djangogirls.org/en/django_models/README.html#creating-a-blog-post-model 
I didn't want to introduce misunderstanding :-)